api/conf/wsgi.py

```diff
@@ -6,20 +6,11 @@
 For more information on this file, see
 https://docs.djangoproject.com/en/4.2/howto/deployment/wsgi/
 """
-
-from gevent import monkey
-
-
-monkey.patch_all()
```

Comment:
We don't use geventlet anymore, so this patching was not necessary and conflicts with `asyncio`.